memcached-image/docker-entrypoint.sh

```diff
@@ -0,0 +1 @@
+submodules/memcached/debian/docker-entrypoint.sh
```

Comment:
My main concern with using entrypoint scripts hosted elsewhere as a general practice is that there is no guarantee that those files are OS distribution agnostic. In some cases (e.g. mariadb), we will end up using the Ubuntu entrypoint script. Then there's the question of how to introduce our own changes into those external files without also inheriting the responsibility of maintaining the corresponding images built from Dockerfiles. I'm not even getting into versioning, that's a proper rabbit hole with its own gravity :)
IMHO, we should either keep our own entrypoint scripts here, or create SUSE specific entrypoint scripts under https://github.com/docker-library. Of course, the latter means that we also need to create corresponding Dockerfiles and properly maintain them.

Reply:
I would
1. try to use the upstream script where possible
1. try to fix the upstream script if needed and possible
1. if 1) and 2) don't work, just copy it (without using a submodule) and maintain our own version

If we realize that 1) and 2) are difficult we can also just maintain our own copies. But I would like to try to use the upstream ones first before doing our own thing.

Reply:
Ok, agreed.